docs/source/public_api/filtsmooth.rst

```diff
@@ -12,17 +12,3 @@ observations into account.
 .. automodapi:: probnum.filtsmooth
     :no-heading:
     :no-main-docstr:
-
-
-probnum.filtsmooth.statespace
-=============================
-
-Probabilistic State Space Models.
-
-This package implements continuous-discrete and discrete-discrete state
-space models, which are the basis for Bayesian filtering and smoothing,
-but also probabilistic ODE solvers.
```

Comment:
What do you think of the following changes of wording?

implements **components of** continuous-discrete and ...

the basis for BFAS, but also **for** probabilistic ODE solvers